tests/pull-test.sh

```diff
@@ -313,6 +313,14 @@ ${CMD_PREFIX} ostree --repo=parentpullrepo rev-parse origin:main > main.txt
 assert_file_has_content main.txt ${rev}
 echo "ok pull specific commit"
 
+repo_init --no-gpg-verify
+${CMD_PREFIX} ostree config --repo=repo set core.payload-link-threshold 0
+${CMD_PREFIX} ostree --repo=repo pull origin main 2>err.txt
```

Comment:
Let's also add a test that verifies running `sha256sum` against a file directly has the same content as the object's payload link?